### PR TITLE
DCS-640 Simplifying routing logic 

### PR DESCRIPTION
**Pull request by andrewrlee**
Always assume edit mode once the report is complete rather than tracking where the user navigated from

### CODE REVIEW

File: integration-tests/integration/reportPages/check-your-answers.spec.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ context('Check your answers page', () => {
     })
   })
 
-  it('Can cancel editing answers from check your answers page ', () => {
+  it.only('Can cancel editing answers from check your answers page ', () => {
```
Comment (JamieDraperUK) [inline integration-tests/integration/reportPages/check-your-answers.spec.js:64]: 👁️

**Review by JamieDraperUK — APPROVED** [review]
lgtm!

File: integration-tests/integration/reportPages/check-your-answers.spec.js (hunk 1 above)
Comment (petergphillips) [reply integration-tests/integration/reportPages/check-your-answers.spec.js:64]: that always catches me out too :grinning:

**Review by JamieDraperUK — APPROVED** [review]
lgtm!

**Pull request merged**